app/controllers/idv/cancellations_controller.rb

```diff
@@ -87,5 +89,10 @@ def session_go_back_path
         idv_session.go_back_path
       end
     end
+
+    def cancel_in_person_enrollment_if_exists
+      return if !IdentityConfig.store.in_person_proofing_enabled
+      current_user.pending_in_person_enrollment&.update(status: :cancelled)
```

Comment:
For most of the flow (except the personal key and "Ready to verify" screens), the enrollment will be in the `establishing` state. Should we capture those here as well?

Reply:
Yeah, good question. What do you think about cancelling all of user's establishing enrollments if they get to this point? And then similarly for the session controller (eg if the user clicks 'Start over')

I don't think that we _need_ to because they'll get cancelled already when the user starts the flow over. But it seems my hygienic to do it sooner rather than waiting for the user to start idv over again

I'm imagining adding a call to this [method](https://github.com/18F/identity-idp/blob/32ec05530763fc2d94865cba5529bb73c96b11ea/app/services/usps_in_person_proofing/enrollment_helper.rb#L69-L74)

Reply:
Added in [5734e3a](https://github.com/18F/identity-idp/pull/6743/commits/5734e3a620830a84757ff839d59ccaef416cb40c)